src/components/cards/mealCard/PEMealCardMobile.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image';
+import { type ReactElement } from 'react';
+import { Icon } from '../../standard/icon/Icon';
+import PeopleEatIcon from '../../standard/icon/PEIcon';
+
+export interface PEMealComponentProps {
+    onClick?: () => void;
+    title?: string;
+    description?: string;
+    picture?: string;
+}
```

Comment:
- The name
- title and description cannot be undefined

```suggestion
export interface PEMealCardProps {
    onClick?: () => void;
    title: string;
    description: string;
    picture?: string;
}
```